test/replication-luatest/gh_8121_memtx_mvcc_replication_stream_test.lua

```diff
@@ -0,0 +1,61 @@
+local t = require('luatest')
+local replica_set = require('luatest.replica_set')
+local server = require('luatest.server')
+
+local g = t.group(nil, t.helpers.matrix({eng = {'memtx', 'vinyl'}}))
```

Comment:
Nit: please don't use confusing abbreviations like 'eng' - 'engine' is short enough.

Reply:
Ok, fixed.